### DISCUSSION TITLE
Metaplex.js File Rename

### DISCUSSION THREAD

**Discussion by TustyBusty** [nftchef/art-engine#32] [Help + Q&A]
Noticed that in line 62, code makes it so that the imgs in the build/solana/img folder will have 1 edition less than the build/img folder. For example 1.png in build/img will become 0.png in solana/build/img. 

Now, I would like my collection to start at Name #0.png. In the config.js file, I put startStart as 1, but that will mean that the first image in build/solana/img will be 0.png. Was wondering if I can change line 62 in metaplex.js to  
`let newEditionCount = editionCountFromFileName` instead?

Was worried in changing this code here as the config.js says to make sure tokenID start matches with smart contract. New to smart contracts, not sure if this will change anything. Thanks for the help!

**Comment by nftchef** [comment]
That is correct. you can change the line. 
Metaplex.js _assumed_ the config.js to be set to 1, and attempts so _reset_ it to zero for Solana (before the start index option existed)

I've added a fix to the Solana util to automatically account for the start index so this is no longer an issue
https://github.com/nftchef/art-engine/pull/39
